locales/eu.json

```diff
@@ -0,0 +1,777 @@
+			"fail": "Elementuaren semantika lehenetsia ez da baliogabetu role bidez =\"none\""
+		},
+		"role-presentation": {
+			"pass": "Elementuaren semantika lehenetsia role =\"presentation\" bidez deuseztatu da.",
+			"fail": "Elementuaren semantika lehenetsia ez da ezeztatu role =\"presentation\" bidez"
+		},
+		"caption-faked": {
+			"pass": "Taula baten lehen lerroa ez da izenburu gisa erabiltzen ('caption')",
+			"fail": "Taularen lehen ilarak izenburu bat izan beharko luke ('caption'), taula-gelaxka baten ordez"
+		},
+		"has-caption": {
+			"pass": "Diseinu-taulak ez du caption atributua erabiltzen",
+			"fail": "Diseinu-taulak caption atributua erabiltzen du"
+		},
+		"has-summary": {
+			"pass": "Diseinu-taulak ez du summary atributua erabiltzen",
+			"fail": "Diseinu-taulak summary atributua erabiltzen du"
+		},
+		"has-th": {
+			"pass": "Diseinu-taulak ez du <th> elementurik erabiltzen ",
+			"fail": "Diseinu-taulak <th> elementuak erabiltzen ditu"
+		},
+		"html5-scope": {
+			"pass": "Scope atributua taulako goiburuen elementuetan bakarrik erabiltzen da (<th>)",
+			"fail": "HTML5en, scope atributuak taula-goiburuen elementuetan bakarrik erabil daitezke (<th>) "
+		},
+		"same-caption-summary": {
+			"pass": "Summary eta <caption> atributuaren edukia ez dago bikoiztuta",
+			"fail": "Summary atributuaren eta <caption> elementuaren edukia berdinak dira"
+		},
+		"scope-value": {
+			"pass": "Scope atributua ondo erabiltzen da",
+			"fail": "Scope atributuaren balioa 'row' edo 'col' bakarrik izan daiteke"
+		},
+		"td-has-header": {
+			"pass": "Hutsik ez dauden datu-gelaxka guztiek taula-goiburuak dituzte",
+			"fail": "Hutsik ez dauden datu-gelaxka batzuek ez dute taula-goibururik"
+		},
+		"td-headers-attr": {
+			"pass": "Headers atributua taulako beste gelaxka batzuei erreferentzia egiteko baino ez da erabiltzen",
+			"fail": "Headers atributua ez da soilik erabiltzen taulako beste gelaxka batzuei erreferentzia egiteko"
+		},
+		"th-has-data-cells": {
+			"pass": "Taulako goiburuen gelaxka guztiak datu-gelaxkak dira.",
+			"fail": "Taulako goiburuen gelaxka guztiak ez dira datu-gelaxkak.",
+			"incomplete": "Taulako datu-gelaxkak ez daude edo hutsik daude"
+		},
+		"hidden-content": {
+			"pass": "Orriaren eduki osoa aztertu da.",
+			"fail": "Arazoak egon ziren orrialde honen edukia aztertzean.",
+			"incomplete": "Orrian ezkutuko edukia dago, baina ez da aztertu. Eduki horren bistaratzea aktibatu beharko duzu, aztertzeko."
+		}
+	},
+	"failureSummaries": {
+		"any": {
+			"failureMessage": "Gorabehera hauetako edozein zuzentzea::{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
+		},
+		"none": {
+			"failureMessage": "Gorabehera hauek zuzentzea (guztiak):{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
+		}
+	},
+	"incompleteFallbackMessage": {
+		"undefined": {
+			"failureMessage": "Gorabehera hauek zuzentzea (guztiak):{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
+		}
+	}
```

Comment:
The `incompleteFallbackMessage` should just be a string instead of an object (I think we had generated it wrongly before). See https://github.com/dequelabs/axe-core/blob/develop/locales/fr.json#L841 for an example. Also, it should be different than the `failureSummaries` message.

The English version of `incompleteFallbackMessage`: 

> axe couldn't tell the reason. Time to break out the element inspector!